react/index.tsx

```diff
@@ -6,54 +6,78 @@ const CSS_HANDLES = ['container'] as const
 
 enum Positions {
   BOTTOM = 'bottom',
+  TOP = 'top',
 }
 
 interface Props {
   position?: Positions
+  verticalSpacing?: number
 }
 
 interface StorefrontComponent extends FC<Props> {
   schema?: any
 }
 
-const StickyLayoutComponent: StorefrontComponent = ({
-  children,
-  position,
-}) => {
-  const container = useRef<HTMLDivElement>(null)
-  const handles = useCssHandles(CSS_HANDLES)
-
-  if (position !== Positions.BOTTOM) {
-    // Only 'bottom' position supported for now!
+const StickyLayoutComponent: StorefrontComponent = ({ children, position, verticalSpacing = 0 }) => {
+  if (!position || (position !== Positions.BOTTOM && position !== Positions.TOP)) {
```

Comment:
You can't do that
https://ru.react.js.org/docs/hooks-rules.html

The number of hooks should be the same on every render.

Reply:
Done.